west.yml

```diff
@@ -74,7 +74,7 @@ manifest:
       revision: 3776c158fe138a72c97c187e4d31c81c37884be9
       path: modules/crypto/mbedtls
     - name: mcuboot
-      revision: c2bd757332bfde11c27019f937b0c61d5e9b9fd9
+      revision: pull/3/head
```

Comment:
Looks good as soon as this is `b976a4c0dcbf5b7bc0908e77a1301520f28813ae` since CI passes.